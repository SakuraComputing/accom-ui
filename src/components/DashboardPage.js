import React from 'react';

const DashboardPage = () =>(
    <body>
        <h1>Acommodation User Interface</h1>
    </body>
);
export default DashboardPage;