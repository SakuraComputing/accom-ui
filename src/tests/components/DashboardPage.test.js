import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import DashboardPage from '../../components/DashboardPage';

describe('Dashboard Page', () => {
    it('should render Expense dashboard page correctly',() => {
        const wrapper = shallow(<DashboardPage />);
        expect(toJSON(wrapper)).toMatchSnapshot();
    });
});
