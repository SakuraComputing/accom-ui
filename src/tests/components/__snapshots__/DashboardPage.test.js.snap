// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Page should render accomodation UI dashboard page correctly 1`] = `
<body>
  <Accommodations />
</body>
`;
